LightFramework/src/rendering/scenerenderer.cpp

```diff
@@ -22,52 +21,97 @@ namespace Light {
 		m_skybox_mesh.reset(VertexArray::create());
 
 		float vertices[] = {
-				//Front
-				-1.0, -1.0, 1.0,
-				1.0, -1.0, 1.0,
-				1.0, 1.0, 1.0,
-				-1.0, 1.0, 1.0,
-				//Left
-				-1.0, -1.0, 1.0,
-				-1.0, 1.0, 1.0,
-				-1.0, 1.0, -1.0,
-				-1.0, -1.0, -1.0,
-				//Right
-				1.0, -1.0, 1.0,
-				1.0, -1.0, -1.0,
-				1.0, 1.0, -1.0,
-				1.0, 1.0, 1.0,
-				//Top
-				-1.0, 1.0, 1.0,
-				1.0, 1.0, 1.0,
-				1.0, 1.0, -1.0,
-				-1.0, 1.0, -1.0,
-				//Bottom
-				-1.0, -1.0, 1.0,
-				-1.0, -1.0, -1.0,
-				1.0, -1.0, -1.0,
-				1.0, -1.0, 1.0,
-				//Back
-				-1.0, -1.0, -1.0,
-				-1.0, 1.0, -1.0,
-				1.0, 1.0, -1.0,
-				1.0, -1.0, -1.0,
+			// Front
+			-1.0,
```

Comment:
It was better the way it was before. One float per line takes up way too much vertical space. Remember, laptop/desktop screens are larger in width than they are in height.